aspnetcore/fundamentals/servers/yarp/direct-forwarding.md

```diff
@@ -39,9 +39,9 @@ See [ReverseProxy.Direct.Sample](https://github.com/microsoft/reverse-proxy/tree
 
 ### Create a new project
 
-Follow the [Getting Started](xref:getting-started) guide to create a project and add the Yarp.ReverseProxy nuget dependency.
+Create a Razor Pages project from the project template and add the [`Yarp.ReverseProxy` NuGet package](https://www.nuget.org/packages/Yarp.ReverseProxy).
```

Comment:
I'd like to avoid this reference to Razor Pages. Do the same steps not work with a Blazor project? Can we just stick with the original text here? 

Or maybe remove the mention of creating a new project and focus instead on adding the package? The section could be renamed "Update dependencies" or something like that.

Reply:
This was merely an in-passing change to get the XREF removed. I'll need to check the whole article see how Razor Pages-centric it is. I'll either fix it or open an issue to address it separately.